spec/models/badge_spec.rb

```diff
@@ -97,4 +97,34 @@
       expect(Badge.display_name('Not In Translations')).to eq('Not In Translations')
     end
   end
+
+  context "PopularLink badge" do
+    it "is awarded for 50 links", focus: true do
+      post = create_post(raw: "https://www.discourse.org/")
+
+      49.times do
```

Comment:
I would prefer we don't create that much records in the tests. Better to lower the badge query expectations.